autoload/codefmt/prettier.vim

```diff
@@ -30,7 +30,9 @@ function! codefmt#prettier#GetFormatter() abort
           \ 'and configure the prettier_executable flag'}
 
   function l:formatter.IsAvailable() abort
-    return executable(s:plugin.Flag('prettier_executable'))
+    let l:cmd = codefmt#formatterhelpers#ResolveFlagToArray(
```

Comment:
Could you send a PR for setting up and using this new helper as a first step? I think it's a nice no-op refactoring which is easier to review if it's not combined with a new behavior change. Some of the diffs are a little harder to read when it's combined like this.

Reply:
Yep, see #138